examples/slack-connector/README.md

```diff
@@ -0,0 +1,190 @@
+# Einstein Bots On Slack
+This repo contains code that creates a link between Slack and Einstein Bots. 
+
+The code is designed to be deployed as a [Heroku](https://dashboard.heroku.com) app.
+
+This README covers details on how to set up the connection between Slack and Einstein Bots.
+
+## Pre-requisites
+
+Before we get started on the setup, the following tooling is necessary
+ 1. Heroku CLI
+    
+    The Heroku CLI provides the best Heroku Developer experience.
+    To install it, run the following command:
+    ```brew tap heroku/brew && brew install heroku```
+ 2. Slack Workspace
+ 
+    You will need to have a workspace where you can set up the Slack Application.
+    This will involve getting in touch with your Slack Admin to provide you with developer access to a workspace where you can install a Slack Application.
+    
+    NOTE: For Salesforce Internal customers, The ```Salesforce Internal``` Workspace is locked down and only available to admins.
+    You will need to instead use the Sandbox Slack to get your application set up, working and tested before you can request it to be moved to ```Salesforce Internal``` workspace.
+    To set this up, 
+    - go to [Aloha](https://aloha.my.salesforce.com)
+    - make sure you have TMP Auth
+    - go to the ```Sandbox Slack``` Tile and select the ```Internal Sanbox``` workspace
+    - once you are in, create a slack channel that you will add your bot to during the installation. You can add the bot to any number of channels later but it needs one fixed channel to be added to during installation.
```

Comment:
This is very salesforce specific. We are open sourcing this, so we don't want to include anything company specific here.

Reply:
Refer to twitter connector for reference.